src/settings/MappingProfiles/MappingProfileDetailsRoute/MappingProfileDetailsRoute.js

```diff
@@ -27,6 +27,7 @@ MappingProfileDetailsRoute.manifest = Object.freeze({
   mappingProfile: {
     type: 'okapi',
     path: 'data-export/mappingProfiles/:{id}',
+    throwErrors: false,
```

Comment:
The same question about disabling errors throwing

Reply:
Answered in the first.